Mathlib/Tactic/ThmLemma.lean

```diff
@@ -0,0 +1,62 @@
+/-
+Copyright (c) 2024 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa
+-/
+import Lean.Elab.Command
+import Lean.Linter.Util
+
+/-!
+
+#  The `theorem` vs `lemma` linter
+
+The usage of `theorem` in `Mathlib` comes with the expectation that the result is "more important"
+than a `lemma`.
+This linter is a step in the direction of enforcing this expectation: if a `theorem`
+does not have a doc-string, then the linter raises a warning.
+-/
+
+open Lean Elab Command
+
+namespace Mathlib.ThmLemma
+
+/-- `thmNoDoc cmd` checks whether the `cmd` is a `theorem` with no doc-string.
```

Comment:
how about "whether the syntax `cmd` represents"?

Reply:
Thanks, I like your suggestion!